partner_communication_switzerland/models/res_partner_zoom_attendee.py

```diff
@@ -85,23 +92,33 @@ def notify_user(self):
                 note=self.optional_message,
                 user_id=user_id)
 
-    def send_confirmation(self):
-        confirmation_config = self.env.ref(
-            "partner_communication_switzerland"
-            ".config_onboarding_zoom_registration_confirmation"
-        )
-        short_notice_config = self.env.ref(
-            "partner_communication_switzerland.config_onboarding_zoom_link")
-        for attendee in self:
-            object_id = attendee.id
-            config = confirmation_config
-            if attendee.zoom_session_id.date_start <= datetime.now() + relativedelta(
-                    days=2):
-                config = short_notice_config
-                object_id = attendee.zoom_session_id.id
-            self.env["partner.communication.job"].create({
-                "partner_id": attendee.partner_id.id,
-                "config_id": config.id,
-                "object_ids": object_id
-            })
-        return True
+    def send_communication(self, config_name):
+        config_id = self.env.ref(config_name.value).id
```

Comment:
Let's make it clear that we should only call this method on a single record.
```suggestion
        self.ensure_one()
        config_id = self.env.ref(config_name.value).id
```